.github/workflows/release.yml

```diff
@@ -0,0 +1,101 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      -
```

Comment:
Just a question. This is legit YAML, but it's less common to put a newline between the "array item" and the contained "map".

I'm down for trying something different to see how it looks/feels.

At this moment, I'm finding it _less_ readable as the dashes (`-`)  seem slightly detached from the content of their step, by my eye... making it distracting for me.

It's one of those strange nits... has zero affect on the correctness, but some kind of meaningful impact to _this_ human.

Reply:
Yeah thats fair. I've changed it to not have a new line after the array item.